ocs_ci/ocs/cluster.py

```diff
@@ -564,6 +571,45 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
+    def get_rebalance_status(self):
+        """
+        This function gets the rebalance status
+
+        Returns:
+            bool: True if rebalance is going on, False otherwise
```

Comment:
I think it makes sense to flip the Statu here, True if Rebalance is complete.

Reply:
Done